<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>464</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="4" column="0">
      <widget class="QLabel" name="label_to">
       <property name="text">
        <string>&amp;To:</string>
       </property>
       <property name="buddy">
        <cstring>lineEdit_to</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_from">
       <property name="text">
        <string>F&amp;rom:</string>
       </property>
       <property name="buddy">
        <cstring>lineEdit_from</cstring>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_carrier">
       <property name="text">
        <string>Carrier:</string>
       </property>
       <property name="buddy">
        <cstring>lineEdit_carrier</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="6">
      <widget class="QTextBrowser" name="textBrowser">
       <property name="font">
        <font>
         <family>仿宋</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="cursor" stdset="0">
        <cursorShape>IBeamCursor</cursorShape>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'仿宋'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="source">
        <url>
         <string>http://www.</string>
        </url>
       </property>
       <property name="openExternalLinks">
        <bool>false</bool>
       </property>
       <property name="openLinks">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="6">
      <widget class="QCalendarWidget" name="calendarWidget">
       <property name="firstDayOfWeek">
        <enum>Qt::Sunday</enum>
       </property>
       <property name="gridVisible">
        <bool>true</bool>
       </property>
       <property name="selectionMode">
        <enum>QCalendarWidget::SingleSelection</enum>
       </property>
       <property name="verticalHeaderFormat">
        <enum>QCalendarWidget::NoVerticalHeader</enum>
       </property>
       <property name="navigationBarVisible">
        <bool>true</bool>
       </property>
       <property name="dateEditEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="pushButton_add">
       <property name="text">
        <string>&amp;ADD</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0" colspan="6">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QToolButton" name="toolButton_color">
       <property name="text">
        <string>&amp;Color</string>
       </property>
       <property name="icon">
        <iconset resource="resource.qrc">
         <normaloff>:/resource/image/textcolor.png</normaloff>:/resource/image/textcolor.png</iconset>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="5" column="5">
      <widget class="QPushButton" name="pushButton_clear">
       <property name="text">
        <string>Cl&amp;ear</string>
       </property>
      </widget>
     </item>
     <item row="4" column="5">
      <widget class="QPushButton" name="pushButton_ok">
       <property name="text">
        <string>&amp;Find</string>
       </property>
      </widget>
     </item>
     <item row="3" column="5">
      <widget class="QComboBox" name="comboBox">
       <item>
        <property name="text">
         <string>all</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>中东运价</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>红海运价</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>地东北非</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>拖车参考</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>报价参考</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="1" colspan="4">
      <widget class="QLineEdit" name="lineEdit_from"/>
     </item>
     <item row="4" column="1" colspan="4">
      <widget class="QLineEdit" name="lineEdit_to"/>
     </item>
     <item row="5" column="1" colspan="4">
      <widget class="QLineEdit" name="lineEdit_carrier"/>
     </item>
     <item row="0" column="5">
      <widget class="QPushButton" name="pushButton_delete">
       <property name="text">
        <string>&amp;Delete</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>历史曲线</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>pushButton_add</tabstop>
  <tabstop>toolButton_color</tabstop>
  <tabstop>pushButton_delete</tabstop>
  <tabstop>calendarWidget</tabstop>
  <tabstop>textBrowser</tabstop>
  <tabstop>lineEdit_from</tabstop>
  <tabstop>lineEdit_to</tabstop>
  <tabstop>lineEdit_carrier</tabstop>
  <tabstop>comboBox</tabstop>
  <tabstop>pushButton_ok</tabstop>
  <tabstop>pushButton_clear</tabstop>
 </tabstops>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
